packages/oui-clipboard/README.md

```diff
@@ -0,0 +1,20 @@
+# oui-clipboard
```

Comment:
We should add this example in the same page as [input-group](https://master.ui-kit.ovh/#!/ovh-ui-kit/input-group).